imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -0,0 +1,30 @@
+"""Methods for decomming packets, processing to level 1A, and writing CDFs for MAG."""
+
+from pathlib import Path
+
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.mag.l0 import decom_mag
+
+
+def mag_l1a(packet_filepath, cdf_directory):
+    """
+    Process MAG L0 data into L1A CDF files at cdf_filepath.
+
+    Parameters
+    ----------
+    packet_filepath:
+        Packet files for processing
+    cdf_directory:
+        Directory for output
+    """
+    mag_l0 = decom_mag.decom_packets(packet_filepath)
+
+    mag_datasets = decom_mag.export_to_xarray(mag_l0)
+
+    write_cdf(mag_datasets, mode="norm", directory=cdf_directory)
```

Comment:
There is no "mode" anymore, so you might need a rebase to get the updates. I think this would be "descriptor" now.

https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/497d446ebfe7a6eea45dc6d85e397338708741e4/imap_processing/cdf/utils.py#L41-L44

Reply:
Yeah, this was a merge fail! Or de-merge? I deleted the commit that did this. I'll update it to match the current version of write_cdf in the code

Reply:
Also, this method signature is changed again in my other CLI PR, just so you're aware.

Reply:
:+1: